src/main/java/com/github/onsdigital/thetrain/App.java

```diff
@@ -131,8 +132,10 @@ public static void registerEndpoints(AppConfiguration cfg) {
 
         registerGetHandler("/transaction", getTransaction(beans), beans.getResponseTransformer());
 
-        registerGetHandler("/contentHash", getContentHash(beans, cfg.isVerifyPublishEnabled()),
-                beans.getResponseTransformer());
+        registerGetHandler("/contentHash", getContentHash(beans, cfg.isVerifyPublishEnabled()), transformer);
```

Comment:
Might be worth updating the line before to use `transformer` instead of `beans.getResponseTransformer())` as well for consistency.